config/config.go

```diff
@@ -1369,9 +1371,20 @@ func getGoGCEnv() int {
 	return DefaultRuntimeConfig.GoGC
 }
 
+type otlpToPrometheusTranslationOption string
+
+var (
+	// KeepUTF8 will keep UTF-8 characters as they are, units and type suffixes will still be added.
+	KeepUTF8 otlpToPrometheusTranslationOption = "KeepUTF8"
```

Comment:
I would prefer enum options that state what the behavior is, even if they are slightly more verbose.  e.g. NoUTF8EscapingWithSuffixes, UnderscoreEscapingWithSuffixes.  "PrometheusRecommendations" seems particularly risky since recommendations can change in the future.

Reply:
That looks better indeed, I struggle to come up with names 😬